### ISSUE TITLE
Broken gradients when using the AMP plugin

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#4529]
## Bug Description

After some very helpful feedback from [Brodie Clark](https://twitter.com/brodieseo) I've found that the AMP plugin is causing issues with gradients.

Here's what I found  out when testing with a linear gradient:

The Web Stories plugin adds a `<div class="page-fullbleed-area">` element which is a child of `<amp-story-grid-layer>`. The gradient is added as inline style to that element.

It will look like this like this:

```css
	background-color: white;
	background-image: linear-gradient(0turn, #fcfcfc 0.43%, #fd174e 100%);
```

The AMP plugin extracts that inline style, gives the `div` a new class(e.g. `amp-wp-c307781`) and adds CSS like `:root:not(#_):not(#_):not(#_):not(#_):not(#_) .amp-wp-c307781{...}` to the `<style amp-custom=>` style. So far so good.

The problem:

The CSS looks as follows (formatted for readability):

```css
:root:not(#_):not(#_):not(#_):not(#_):not(#_) .amp-wp-c307781 {
	background-color: white;
	background-image: linear-gradient(0 turn,#fcfcfc .43%,#fd174e 100%);
}
```

Notice how `0turn` became `0 turn`.

This makes the `background-image` declaration now invalid CSS.

**Note:**

**This seems to apply only to linear gradients.**

Radial gradients work fine, as the CSS looks like `background-image:radial-gradient(#fcfcfc 0.43%, #fd174e 100%)` and the AMP plugin does not break it.

## Expected Behaviour

Gradients should work just fine when the AMP plugin is active.

## Steps to Reproduce

1. Activate AMP plugin
1. Create a new story
1. Add a gradient to the page
1. Save story and view it on the frontend

## Screenshots

Expectation:
<img width="367" alt="Screenshot 2020-09-17 at 12 07 44" src="https://user-images.githubusercontent.com/841956/93456758-6d653980-f8de-11ea-9bf7-0e849132cc2e.png">

vs. reality:

<img width="525" alt="Screenshot 2020-09-17 at 12 07 48" src="https://user-images.githubusercontent.com/841956/93456748-6c340c80-f8de-11ea-95ed-d459b6d46def.png">

## Additional Context

- Plugin Version: 1.0.0-rc.1 + AMP plugin 2.0.2
- Operating System: macOS
- Browser: Chrome 85

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
Known issue in the AMP plugin: https://github.com/ampproject/amp-wp/issues/4604
Upstream PR: https://github.com/sabberworm/PHP-CSS-Parser/pull/193

cc @westonruter

**Comment by westonruter** [comment]
We'll try to get this patched for v2.0.2 as well.

**Comment by westonruter** [comment]
AMP plugin [v2.0.2](https://github.com/ampproject/amp-wp/releases/tag/2.0.2) has been release, so this issue is fixed. See screenshots of before/after: https://github.com/ampproject/amp-wp/pull/5392#issuecomment-694539428.
